mysql-async/src/test/java/com/github/jasync/sql/db/mysql/BinaryColumnsSpec.kt

```diff
@@ -8,20 +8,21 @@ import java.util.*
 import org.junit.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertNotNull
+import org.junit.Assert.assertArrayEquals;
 
 class BinaryColumnsSpec : ConnectionHelper() {
 
     @Test
     fun `correctly load fields as byte arrays`() {
 
         val create = """CREATE TEMPORARY TABLE t (
-                     |  id BIGINT NOT NULL AUTO_INCREMENT,
-                     |  uuid BINARY(36) NOT NULL,
-                     |  address VARBINARY(16) NOT NULL,
-                     |  PRIMARY KEY (id),
-                     |  INDEX idx_t_uuid (uuid),
-                     |  INDEX idx_t_address (address)
-                     |);"""
+                       id BIGINT NOT NULL AUTO_INCREMENT,
```

Comment:
I think you should leave the pipe here and use trim indent: 
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-margin.html